web/templates/settings/preferences_information.hbs

```diff
@@ -29,6 +29,22 @@
                             <span class="status-text">{{t "Working remotely" }}</span>
                         </div>
                         {{/if}}
+                        {{#if (eq this.code 3)}}
+                        <div class="profile-with-avatar">
+                            <div class="user-profile-picture">
+                                {{!-- TODO(evy): figure out why it can't find this --}}
+                                {{!-- <img src="{{../profile_picture}}"/> --}}
```

Comment:
looking for help here!

Reply:
This is called from settings/preferences.hbs ← settings/user_preferences.hbs ← settings_tab.hbs ← `settings.build_page`, which sets `profile_picture`, but also from settings/preferences.hbs ← settings/organization_user_settings_defaults.hbs ← settings/admin_tab.hbs ←`admin.build_page`, which does not.

Reply:
Thanks so much